### ISSUE TITLE
End Result Predictor in TIC TAC TOE Game

### ISSUE THREAD

**Issue by akash435** [Tejas1510/Hacking-Scripts#645]
In this script , we allow user to fill the TIC TAC TOE board for with 'X', 'O' and 'empty space'
and there will be a guess button, clicking on which gives the end result of game, like who wins, who lose, or it's a tie.
This script also check that the way user has filled the TIC TAC TOE board, is it possible in real life TIC TAC TOE game.

Also for this, on the starting of the game, it will be clearly given that either person with 'X' will start or person with 'O' will start.

So basically this game is about, predicting the result of the game from any intermediate step (if valid), otherwise if the intermediate state that user has filled is not valid, then the script will also lets us know that also.

**Comment by akash435** [comment]
@Tejas1510 @RaveenaBhasin @GowthamGoush , please assign me this under LGM SOC, can add this script of predicting the  end result of TIC TAC TOE, if we are give any intermediate state.

**Comment by RaveenaBhasin** [comment]
@akash435 Go ahead!
